mobile/screens/Point_of_interest_info_page.js

```diff
@@ -0,0 +1,56 @@
+import { styles } from "../styles/basic_styles";
```

Comment:
Dit bestand is bij mij nog steeds met een hoofdletter dus geeft hij een error in de app.js